### DISCUSSION TITLE
sourcemap不能正确加载

### DISCUSSION THREAD

**Discussion by uncledu** [NervJS/taro#16772] [Q & A]
我的taro项目，不久前开始sourcemap不对了，微信开发者工具报错
DevTools failed to load SourceMap: Could not load content for weapp:///http:/127.0.0.1:32479/appservice/vendors.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME

实际上这个sourcemap文件是能够通过http://127.0.0.1:32479/appservice/vendors.js.map 访问到的
尝试搜索了一番，没有找到解决方法，哪位大佬可以指点一下迷津

**Comment by Roderick-cc** [comment]
切换一下调试的基础库版本
